src/main/java/kubatech/loaders/MobRecipeLoader.java

```diff
@@ -92,12 +93,8 @@ public class MobRecipeLoader {
 
     private static final Logger LOG = LogManager.getLogger(Tags.MODID + "[Mob Recipe Loader]");
 
-    private static final String dropFewItemsName = isDeobfuscatedEnvironment ? "dropFewItems" : "func_70628_a";
-    private static final String dropRareDropName = isDeobfuscatedEnvironment ? "dropRareDrop" : "func_70600_l";
-    private static final String setSlimeSizeName = isDeobfuscatedEnvironment ? "setSlimeSize" : "func_70799_a";
     private static final String addRandomArmorName = isDeobfuscatedEnvironment ? "addRandomArmor" : "func_82164_bB";
     private static final String enchantEquipmentName = isDeobfuscatedEnvironment ? "enchantEquipment" : "func_82162_bC";
```

Comment:
forgot to remove?

Reply:
Still used to determine if the  mob class is overriding them https://github.com/GTNewHorizons/KubaTech/blob/b91f9ba5a7d1e32ad11c4d7499882dfb7cbd002f/src/main/java/kubatech/loaders/MobRecipeLoader.java#L844-L855

Reply:
oh I see